.github/workflows/rust.yml

```diff
@@ -20,10 +20,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Add clippy
-        uses: actions-rs/toolchain@v1.0.6
+        uses: dtolnay/rust-toolchain@master
         with:
-          profile: minimal
-          default: true
+          toolchain: 1.68
```

Comment:
The `actions-rs/toolchain` read version from `rust-toolchain` if no toolchain input was specified. However, `dtolnay/rust-toolchain`does not have this feature. And the toolchain version for `dtolnay/rust-toolchain@1.68` is set to 1.68.2, not 1.68.

Reply:
@ever0de The output `name` of `dtolnay/rust-toolchain` is the same as the input `toolchain`. How about reading the value of `rust-toolchain` and putting it as input like below?

```yaml
- name: Read rust-toolchain file
  id: rust-toolchain
  run: echo "toolchain=$(cat rust-toolchain)" >> $GITHUB_OUTPUT
- name: Add clippy
  uses: dtolnay/rust-toolchain@master
  with:
    toolchain: ${{ steps.rust-toolchain.outputs.toolchain }}
    components: clippy
```

Reply:
Oh yeah, that's good, I'm in favor of that as well.  Can you just test that it reads fine in CI?

Reply:
@ever0de Sure :)